_posts/2021-07-19-dd_00002_transistor.md

```diff
@@ -0,0 +1,236 @@
+
+Here you have it: A transistor is basically a tiny faucet to control the flow of
+electrons! Unlike a real faucet, which can be opened in one quarter, one half,
+and so on, we operate transistors in only two _states_[^1]: ON or OFF. Because
+engineers are lazy, we like to abbreviate the states with 1 and 0, respectively.
+And this is precisely the reason why computers "think" in 1s and 0s (also called
+_binary_)! We will uncover the secrets of how this thinking actually works in
+our upcoming posts.
+
+<!--
+todo:
+
+- halb offen gibts auch bei transistoren ✔
+- nicht jeder transistor wird über eine spannung gesteuert
+
+[todo] Just for completeness: there are also transistors that use a current flow
+instead of a voltage to control
+
+-->
+
+Next, let us have a look at a concrete type of transistor.
+
+[^1]:
+    Actually, transistors can also be "half-open". However, this operating mode
+    is out of interest in _digital_ systems. If you are curious: transistors
+    operated in states between "fully off" and "fully on" are commonly found in
+    _analog_ circuits, such as amplifiers.
+
+## The MOSFET
+
+There are two types of transistors out there. One are the **field-effect
+transistors**, or FETs, and the other ones are the **bipolar transistors**. Both
+types have the same goal, namely to control the flow of some current. However,
+even though both are semiconductor devices, they work based on different
+principles. This makes each of them useful in special application areas. As
+nowadays in digital microchips solely FETs are used, we will focus our
+discussion on them for now.
+
+Let's look at the most basic FET: the **MOSFET**. Its name stands for
+_metal-oxide-semiconductor field-effect transistor_. The next picture shows a
+cross-section of a MOSFET:
+
+<figure class="img-center" style="width:50%">
+  <img src="/assets/images/dd_00002-mosfet-cross-section.png" alt="">
+  <!-- <figcaption class="figure-caption text-center">Microchips</figcaption> -->
+</figure>
+
+You might have noticed the same three terminals (gate, source, and drain) we
+introduced earlier, this time with their common abbreviations (G, S, D), as they
+are used by engineers. "B" stands for **bulk** (sometimes also called _body_ or
+_substrate_). The bulk is basically the basis on top of which the transistor is
+built (if you remember from the previous post this basis is nothing but the
+actual _wafer_). Bulk, source, and drain are made from **silicon**. The gate on
+the top can theoretically be made of any metal, usually, poly-silicon is used
+today. Between the gate "plate" and the rest of the transistor, we can see
+another layer called **oxide**. The role of this oxide layer is to be an
+_insulating_ layer so that no current can flow between the gate and the bulk (or
+source/drain).
+
+When we apply a voltage at the gate terminal, a conducting **channel** starts to
+form between the source and the drain (see picture above). This channel then
```

Comment:
I would leave the "(see picture above)". I was expecting at that moment that I can see how the channel forms. The picture, however, only shows where. Might be confusing